benchmarks/koa-isomorphic-router.js

```diff
@@ -1,11 +1,12 @@
 'use strict'
 
 const Koa = require('koa')
-const router = require('koa-isomorphic-router')()
+const Router = require('koa-isomorphic-router')
 
 const app = new Koa()
+const router = new Router()
 
-router.get('/', function (ctx) {
+router.get('/', async function (ctx) {
```

Comment:
Would not it be faster the `function (ctx, next){}` syntax?

Reply:
It's typo 😄